gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -653,6 +659,30 @@ const CssVariablesPage = () => {
             <ColorDocRow colorVar="--nx-color-policy-threat-slider-text">
               The default text color of <NxCode>NxPolicyThreatSlider</NxCode>.
             </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-radio-checkbox-selected">
+              The default selected color for radios and checkboxes.
+            </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-radio-disabled-border">
+              The default border color for disabled <NxCode>NxRadio</NxCode>.
+            </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-radio-disabled-box-shadow">
```

Comment:
I feel like for these the the fact that they're box shadows is very much an implementation detail.  We might want to avoid expressing that in the variable name.

Reply:
> I feel like for these the the fact that they're box shadows is very much an implementation detail. We might want to avoid expressing that in the variable name.

Let me know your thoughts! 29f54f813d110ac8ff28b80f676350b3d9feed62

Reply:
> Let me know your thoughts! https://github.com/sonatype/sonatype-react-shared-components/commit/29f54f813d110ac8ff28b80f676350b3d9feed62

:thinking: It's awkward to name because the construction of the disabled radios doesn't match the non-disabled, and also doesn't match between light and dark mode.  The naming of the dot in the center of the disabled+checked state is also in play, as you have that currently named `background`.  To me, this "inner border" as you call it is actually more conceptually the background. What if we call the dot `dot`, call this middle section the `background`, and call the border `border`.  All of these are aligned more to what they look like than to how they're implemented, but I think that's the right choice here since the implementation is pretty funky and subject to change in the future.

Reply:
> > Let me know your thoughts! [29f54f8](https://github.com/sonatype/sonatype-react-shared-components/commit/29f54f813d110ac8ff28b80f676350b3d9feed62)
> 
> 🤔 It's awkward to name because the construction of the disabled radios doesn't match the non-disabled, and also doesn't match between light and dark mode. The naming of the dot in the center of the disabled+checked state is also in play, as you have that currently named `background`. To me, this "inner border" as you call it is actually more conceptually the background. What if we call the dot `dot`, call this middle section the `background`, and call the border `border`? All of these are aligned more to what they look like than to how they're implemented, but I think that's the right choice here since the implementation is pretty funky and subject to change in the future.

Renamed to `ring` and `dot`: 693777f1b65e65acda79a9816bccbc90a0c8e248